pages/templatetags/pages_tags.py

```diff
@@ -27,6 +27,17 @@ def candidates_url():
 def endorsement_process_url():
     return settings.ENDORSEMENT_PROCESS_URL
 
+@register.simple_tag
+def get_alert_level_class(value):
+    """Pass in alert level value and get back approporiate CSS class."""
+    classes_dict = {
+        1: 'success',
+        2: 'info',
+        3: 'warning',
```

Comment:
Good call. For second point I'm going to leave just as `info` `warning` etc. because they can be used with other bootstrap classes should we ever need them, not just alerts.